### ISSUE TITLE
Win10 compiled not showing globe

### ISSUE THREAD

**Issue by lp59lp** [GitSquared/edex-ui#282]
![edexui-typeerror](https://user-images.githubusercontent.com/2379608/48969435-b76b8580-efbb-11e8-803f-18765c010a9f.PNG)

**Comment by GitSquared** [comment]
Duplicate of #217 - i'm waiting on an upstream release that will patch this, should be coming in a few days.
